### DISCUSSION TITLE
OTP Read/Write

### DISCUSSION THREAD

**Discussion by mirkomatonti** [stlink-org/stlink#1351] [Q&A (resolved)] [answered]
Hello,

I need to read and write the otp area for a project of mine. I noticed that this feature has not being implemented yet so I did a quick implementation and it works. I know that OTP area is not standard across the STM32 line so how would that work in case I want to do a PR? Can I just add support to L4x? (which is pretty easy as the OTP area is treated as FLASH) or do I need to do an implementation for all the supported boards?

**Comment by Nightwalker-87** [comment, marked as answer]
Topic addressed in issue #1352.
